stage #1/module #1. Exceptions/content.md

```diff
@@ -1,20 +1,175 @@
-# How to write your own exception classes?
+# Exceptions
```

Comment:
Maybe it's worth splitting this module into several lessons?

Reply:
I don't know, what do others think?